Meta/Websites/man.serenityos.org/cant-run-application.md

```diff
@@ -1,4 +1,6 @@
-# Sorry :^(
```

Comment:
Why the removal of the sad caret emoji? :^(

Reply:
Because I've seen the community take the piss out of the trend of cutesy or humorous 404 pages and thought this might count.
But I can add it back 😄